scripts/remote_diff/show_neard_version.py

```diff
@@ -0,0 +1,22 @@
+import sys
```

Comment:
Please add a command explaining the purpose of this script and an example of its intended use.